spec/lib/compiler_spec.rb

```diff
@@ -307,6 +307,14 @@
     end
   end
 
+  describe 'x-strings' do
+    it 'removes the trailing semicolon' do
+      compiler = Opal::Compiler.new('a = `1;`')
+      expect(compiler).to receive(:warning).once
+      expect(compiler.compile).not_to include(";)")
+    end
+  end
+
```

Comment:
should also test for multiline xstr, possibly with multiple semicolons within them, especially before newlines